### PR TITLE
German translation

### PR DESCRIPTION
**Pull request by vaelund**

### CODE REVIEW

**Comment by 12345ieee** [comment]
The travis build failed with:
```
de.po:515: keyword "Pokefinder" unknown
de.po:515:43: syntax error
```
you have some error there.

**Comment by sriemer** [comment]
Would be better with merging 83b7621 and 8d72b3c. It is strange to introduce Italian translation in the German localization.

**Comment by 12345ieee** [comment]
Eh, let @vaelund add as many commits as he wishes, I'll squash before merging.

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+msgid "Target address here (CTRL+L)"
+msgstr "Zieladresse auswählen (CTRL+L)"
+
+#: ../gui/GameConqueror.ui.h:4
+msgid "Enter address to view"
+msgstr "Adresse zur Anzeige eingeben"
+
+#: ../gui/GameConqueror.ui.h:5
+msgid "Jump to address (CTRL+ENTER)"
+msgstr "Springe zu Adresse (CTRL+INVIO)"
```
Comment (sriemer) [inline po/de.po]: What is CTRL+INVIO?
My keyboard knows STRG+ENTER.

**Comment by sriemer** [comment]
Locking is not "überwachen" in German. IMHO "locking" is even the wrong word in English. It should be "freeze" and translates to "einfrieren" (variable is held at a certain value by cyclic updating to the previous/user value).

**Comment by vaelund** [comment]
Thanks for checking my work.
Anything else i can improve?

**Comment by 12345ieee** [comment]
Travis approves, and layout-wise everything seems ok.
I'll leave the last word on the translation itself to @sriemer , because I don't speak German.

Thanks for contributing, @vaelund .

**Comment by 12345ieee** [comment]
Sebastian had no further comments, so I went ahead and merged it as a31c2e5 .
Closing this, thanks for contributing.

**Pull request closed without merging**